pipeline_steps/checkmarx.groovy

```diff
@@ -68,7 +73,10 @@ def scan(String scan_type, String repo_name, String exclude_folders){
                 )
             } catch (Exception e){
                 print ("Caught exception while running checkmarx scan: "+e)
-                sleep(time: 30, unit: "SECONDS")
+                sleep(time: waitTime, unit: "SECONDS")
```

Comment:
I think this would re-introduce the problem that Hugh mentioned before, so the waitTime would get to be too large.

Reply:
I believe I did - the initial value is 8 seconds (instead of 30 seconds) - we ramp up from there.